migration/config-repair/remove_incorrect_configs.py

```diff
@@ -0,0 +1,20 @@
+import tables
+from shutil import copyfile
+
+
+def remove_configs(filename, cluster, stationnumber):
+    """
+    Remove all but the first configuration row
+    """
+    
+    copyfile(filename, filename+'_backup')
+
+    with tables.open_file(filename, "a") as data:
+        config = data.get_node("/hisparc/cluster_"+cluster+"/station_"+str(stationnumber)+"/config")
+        print len(config)
+        config.remove_rows(1,len(config))
```

Comment:
The end point for [`remove_rows`](http://www.pytables.org/usersguide/libref/structured_storage.html#tables.Table.remove_rows) is optional
> If None all rows after start will be removed.